lib/internal/nsolid_diag.js

```
@@ -1,11 +1,17 @@
 'use strict';
 
+const {
+  Number,
```

Biome (2.4.6)

[error] Do not shadow the global "Number" property.

(lint/suspicious/noShadowRestrictedNames)